helm/values.yaml

```diff
@@ -46,8 +46,19 @@ extraLabels: {}
 deployment:
   # -- give a few seconds of grace time on shutdown to allow queries to finish
   terminationGracePeriodSeconds: 5
+  # -- Controls storage mode. When true, persistent storage via volumeClaimTemplates (PVC) is used
+  # -- and `deployment.storageSpec` is applied. When false, PVCs are not created and
+  # -- `deployment.storageHostPath` is used instead. Only one mode is active at a time.
+  persistantStorageEnabled: true
```

Comment:
I think this name might be confusing :thinking: when storage host path is used, it is still persistent, just not on a volume. It will persist on the host.